views/blocks/wirecard_credit_card_fields.tpl

```diff
@@ -31,14 +32,24 @@
       <input type="hidden" name="challenge" value="[{$challenge}]">
       <input type="hidden" name="sDeliveryAddressMD5" value="[{$oView->getDeliveryAddressMD5()}]">
       [{if $oView->isActive('PsLogin') || !$oView->isConfirmAGBActive()}]
-    <input type="hidden" name="ord_agb" value="1">
+        <input type="hidden" name="ord_agb" value="1">
       [{else}]
-    <input type="hidden" name="ord_agb" value="0">
+        <input type="hidden" name="ord_agb" value="0">
       [{/if}]
       <input type="hidden" name="oxdownloadableproductsagreement" value="0">
       <input type="hidden" name="oxserviceproductsagreement" value="0">
       <div id="creditcard-form-div"></div>
+
+      [{if $user->hasAccount()}]
+        <div style="margin-bottom: 10px">
+          <input type="checkbox" name="save_checkbox" id="save-checkbox"/>
```

Comment:
If you put the input element inside the label, you can omit the `id` attribute. This might be useful to avoid ID conflicts (`save-checkbox` could already be taken).

```html
<label>
  <input type="checkbox" name="save_checkbox"/> [{oxmultilang ident="wd_vault_save_text"}]
</label>
```